internal/sql/types/row.go

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package types
+
+import (
+	"fmt"
+
+	"github.com/hazelcast/hazelcast-go-client/sql"
+)
+
+var ErrIndexOutOfRange = fmt.Errorf("index out of range")
+var ErrColumnNotFound = fmt.Errorf("column not found")
```

Comment:
Do you think these sentinel errors are useful to the user?

Reply:
Definitely not since they are in the internal package. I guess move them to /sql package to export or make them inline errors.

Not sure what to do though. What do you think?